src/patternfly/components/Wizard/wizard.scss

```diff
@@ -27,122 +28,116 @@
   }
 
   // Close
-  --#{$wizard}__close--Top: calc(var(--#{$pf-global}--spacer--lg) - var(--#{$pf-global}--spacer--form-element));
+  --#{$wizard}__close--Top: calc(var(--pf-t--global--spacer--lg) - var(--#{$pf-global}--spacer--form-element));
   --#{$wizard}__close--Right: 0;
```

Comment:
Would we want to update this from 0? Right now at < xl viewports the button is right up against the edge of the wizard header, which it may just be the context of the example as it would look fine in the context of an actual page most likely.

![image](https://github.com/patternfly/patternfly/assets/70952936/5dc2955a-58c9-4e38-8ac7-749288fd3aaf)